src/all/akuma/src/eu/kanade/tachiyomi/extension/all/akuma/Akuma.kt

```diff
@@ -38,10 +48,17 @@ class Akuma : ParsedHttpSource() {
 
     private val ddosGuardIntercept = DDosGuardInterceptor(network.client)
 
+    private val preferences: SharedPreferences by lazy {
+        Injekt.get<Application>().getSharedPreferences("source_$id", 0x0000)
+    }
+
+    private var iconified = preferences.getBoolean(PREF_TAG_GENDER_ICON, false)
+    private var rateLimit = preferences.getString(RATE_LIMIT, "2")?.toIntOrNull() ?: 2
```

Comment:
dont add rate limit preference

Reply:
May I ask why? Some of the others do this.

Reply:
generally we don't want to make it configurable for users, rate limits are there for a reason.